page-objects/pages/HMWRKHomeworkListPage.js

```diff
@@ -99,7 +99,7 @@ module.exports = {
         return isTaskOnList;
     },
 
-    userFindsTheTask: async function (taskname) {
+    getTaskFromList: async function (taskname) {
```

Comment:
It is not "get"  its click (there is no return). Maybe I am wrong but it is not a duplicate of  clickOnTask(). But this method looks better. It has more checks.

Reply:
there is a diffrent method but renamed it. done